terraform/lambda/credential-issuer.tf

```diff
@@ -10,4 +10,8 @@ module "credential-issuer" {
   handler                = "uk.gov.di.ipv.lambda.CredentialIssuerHandler::handleRequest"
   function_name          = "${var.environment}-credential-issuer"
   role_name              = "${var.environment}-credential-issuer-role"
+
+  allow_access_to_user_issued_credentials_table = true
+  user_issued_credentials_table_policy_arn      = aws_iam_policy.access-user-issued-credentials-table.arn
```

Comment:
A change has been made to the terraform dynamo db policy names. So the value of this should now be 

aws_iam_policy.policy-user-issued-credentials-table.arn